### PR TITLE
Add documentation about how to use the grafana UI

### PR DESCRIPTION
**Pull request by ycointe**
**Component**:

**Context**: Add documentation

**Summary**: How to reach the grafana UI

**Acceptance criteria**: Be able to reach the Grafana UI

---

Closes:

### CODE REVIEW

File: docs/grafana-ui.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Grafana-UI
+==========
+
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using a operationnal cluster.
+
+  On your host, you will need to install kubectl.
```
Comment (gdemonet) [inline docs/grafana-ui.rst]: That shouldn't be part of the `todo` note, IIUC. Instead, write it as a requirement, either in plain text, or in a `.. warning::` directive.

File: docs/grafana-ui.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+On your host, connect to the vagran VM
```
Comment (gdemonet) [inline docs/grafana-ui.rst]: Same remark as for #1182 - this documentation isn't targetting Vagrant deployments. We may mention it in some situations, but it shouldn't be as obvious as done here.

Comment (gdemonet) [reply docs/grafana-ui.rst]: If I may, here's a suggestion for replacing this procedure:

```
You will first need ``kubectl`` installed on your host.

To authenticate with the cluster, first retrieve the admin kubeconfig on your host:

.. code-block:: shell

   $ scp root@bootstrap:/etc/kubernetes/admin.conf ./admin.conf

Using this file, activate port-forwarding for Grafana on your host:

.. code-block:: shell

   $ kubectl -n monitoring --kubeconfig ./admin.conf port-forward svc/grafana 3000

Now, you can navigate to ``http://localhost:3000`` on your host, which should display Grafana UI.
```

**Review by gdemonet — DISMISSED** [review]
The procedure idea is good, now it needs to be revamped to take into account that Vagrant is not a standard deployment, and to simplify some parts (I suggested a replacement from what you did).

File: docs/grafana-ui.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
+Grafana-UI
+==========
+
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
```
Comment (slaperche-scality) [inline docs/grafana-ui.rst]: s/vagrant/Vagrant/

File: docs/grafana-ui.rst (hunk 2 above)
Comment (slaperche-scality) [inline docs/grafana-ui.rst]: missing `t` at `Vagrant` (also capitalize the first `v`).

File: docs/grafana-ui.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+  $ cp /etc/kubernetes/admin.conf /vagrant
+
+Return on you host then forward the port used by Grafana
+
+.. code-block:: shell
+
+  kubectl --namespace monitoring port-forward svc/grafana 3000
+
+Then open your web browser and navigate on ``http://localhost:3000``
```
Comment (slaperche-scality) [inline docs/grafana-ui.rst]: s/on/to/ ?

File: docs/grafana-ui.rst (hunk 1 above)
Comment (ycointe) [reply docs/grafana-ui.rst]: modified. I tried to encompass all the different infrastructure. (I still specified that we use vagrant)

File: docs/grafana-ui.rst (hunk 2 above)
Comment (ycointe) [reply docs/grafana-ui.rst]: correction applied.

File: docs/grafana-ui.rst (hunk 3 above)
Comment (ycointe) [reply docs/grafana-ui.rst]: correction done.

File: docs/grafana-ui.rst (hunk 2 above)
Comment (ycointe) [reply docs/grafana-ui.rst]: correction done.

File: docs/grafana-ui.rst (hunk 4 above)
Comment (ycointe) [reply docs/grafana-ui.rst]: correction done.

File: docs/grafana-ui.rst (hunk 3 above)
Comment (slaperche-scality) [reply docs/grafana-ui.rst]: Where?

**Comment by ycointe** [comment]
@bert-e reset

File: docs/grafana-ui.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+.. warning::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using a operationnal cluster.
+
+
+You will first need the lastest ``kubectl`` version installed on your host.
+
+From te bootstrap node, get the port used by Grafana:
```
Comment (jbertran) [inline docs/grafana-ui.rst]: `s/te/the`

File: docs/grafana-ui.rst [hunk 6]
```diff
@@ -0,0 +1,36 @@
+Grafana UI
+==========
+
+
+.. warning::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using a operationnal cluster.
```
Comment (jbertran) [inline docs/grafana-ui.rst]:
```suggestion
   all requirements for reaching the UI using a operational cluster.
```

File: docs/grafana-ui.rst [hunk 7]
```diff
@@ -0,0 +1,36 @@
+Grafana UI
+==========
+
+
+.. warning::
+
+  The following test has been realized on vagrant only - One should list
```
Comment (jbertran) [inline docs/grafana-ui.rst]: Should be indented with 3 spaces, not 2

File: docs/grafana-ui.rst [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  all requirements for reaching the UI using a operationnal cluster.
+
+
+You will first need the lastest ``kubectl`` version installed on your host.
+
+From te bootstrap node, get the port used by Grafana:
+
+.. code-block:: shell
+
+  $ kubectl --kubeconfig=/etc/kubernetes/admin.conf get service grafana -n monitoring
```
Comment (jbertran) [inline docs/grafana-ui.rst]: ditto spaces

File: docs/grafana-ui.rst [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+  NAME      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+  grafana   ClusterIP   10.109.125.193   <none>        3000/TCP   1h
+
+Please note the port used by Grafana (here ``3000``)
+
+To authenticate with the cluster, retrieve the admin kubeconfig on your host:
+
+.. code-block:: shell
+
+  $ scp root@bootstrap:/etc/kubernetes/admin.conf ./admin.conf
```
Comment (jbertran) [inline docs/grafana-ui.rst]: ditto spaces

File: docs/grafana-ui.rst [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+Return on you host then forward the port used by Grafana
+
+.. code-block:: shell
+
+  kubectl --namespace monitoring port-forward svc/grafana 3000
```
Comment (jbertran) [inline docs/grafana-ui.rst]: ditto spaces

File: docs/grafana-ui.rst [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+Please note the port used by Grafana (here ``3000``)
+
+To authenticate with the cluster, retrieve the admin kubeconfig on your host:
+
+.. code-block:: shell
+
+  $ scp root@bootstrap:/etc/kubernetes/admin.conf ./admin.conf
+
+Return on you host then forward the port used by Grafana
```
Comment (jbertran) [inline docs/grafana-ui.rst]:
```suggestion
Return on your host and forward the port used by Grafana
```

**Review by jbertran — COMMENTED** [review]
Nice.

We could benefit from having a convention to specify on what machine we are - I've already done so in the hostname doc by having `[<machine>]$` as the prompt.

File: docs/grafana-ui.rst (hunk 5 above)
Comment (ycointe) [reply docs/grafana-ui.rst]: done

File: docs/grafana-ui.rst (hunk 6 above)
Comment (ycointe) [reply docs/grafana-ui.rst]: done

**Comment by ycointe** [comment]
@bert-e  reset

**Comment by ycointe** [comment]
@bert-e approve

**Comment by jbertran** [comment]
@bert-e approve

**Review by jbertran — APPROVED** [review]

**Review by jbertran — APPROVED** [review]

**Pull request merged**